media/css/firefox/firstrun/variants.less

```diff
@@ -0,0 +1,162 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+/* ------------------------------------
+ * CSS styles for firstrun variant e
+ --------------------------------------- */
+.variant-e {
+
+    .rightcol {
+        background: none;
+        margin-top: 0;
+        width: 46%;
+
+        h1 {
+            margin-top: -10px;
+            margin-bottom: 10px;
+        }
+
+        p {
+            .font-size(1.1rem);
+            .open-sans-light;
+        }
+    }
+
+    .leftcol {
+        background-color: #fff;
+        padding: @baseLine 35px 20px @baseLine;
+        border-radius: 6px;
+        width: 40%;
+    }
+
+    @media only screen and (max-width: 800px) {
```

Comment:
Suggest setting this breakpoint at `max-width: @widthDesktop` so it kicks in before we hit 800px. Otherwise we get horizontal scrolling and cutoff text anywhere between 1000px and 800px.